app/javascript/styles/_scatterPlot.scss

```diff
@@ -0,0 +1,80 @@
+
+.plot-container.plotly {
+  float: left;
+}
+
+.scatter-graph {
+  margin-left: 40px;
+  display: flex;
+}
+
+.scatter-legend {
+  float: right;
+  overflow: scroll;
```

Comment:
While having the visual indication that the legend can scroll is nice, when it doesn't need to scroll, setting `overflow: scroll` can have weird behavior depending on browser/OS.  I noticed in your video that the scrollbars don't show, but on my machine they are visible the entire time, even when not needed.  This persists both on my external monitor and laptop screen (even when the external is disconnected).

![Screen Shot 2021-11-08 at 10 09 22 AM](https://user-images.githubusercontent.com/729968/140767725-e9859a1c-4518-4055-a437-1d5e98970960.png)

Reply:
Could you try again?

Thanks for demo-ing the issue for me.  Devon suggested `overflow: auto`, and I've updated accordingly.

I suspect you have “Show scroll bars” set to “Always” in System Preferences -> General.  I just updated my preferences to that, and could reproduce the issue, and confirmed `overflow: auto` fixes it for me.  [Here's a detailed exploration](https://kilianvalkhof.com/2021/css-html/you-want-overflow-auto-not-overflow-scroll/) that helped me reliably reproduce what you saw.

Reply:
Auto works!  Totally solved the issue; scrollbars only show when needed.

Reply:
Also, my scrollbar preference is set to `Automatically based on mouse or trackpad`